specifyweb/specify/migrations/0005_stratigraphy_age.py

```diff
@@ -0,0 +1,374 @@
+                ('ordinal', models.IntegerField(blank=True, db_column='Ordinal', null=True)),
+                ('remarks', models.TextField(blank=True, db_column='Remarks', null=True)),
+                ('timestampcreated', models.DateTimeField(db_column='TimestampCreated', default=django.utils.timezone.now)),
+                ('timestampmodified', models.DateTimeField(blank=True, db_column='TimestampModified', default=django.utils.timezone.now, null=True)),
+                ('version', models.IntegerField(blank=True, db_column='Version', default=0, null=True)),
+                #relationships
+                ('attachment', models.ForeignKey(db_column='AttachmentID', on_delete=specifyweb.specify.models.protect_with_blockers, related_name='relativeageattachments', to='specify.attachment')),
+                ('collectionmember', models.ForeignKey(db_column='CollectionMemberID', on_delete=specifyweb.specify.models.protect_with_blockers, related_name='relativegeattachments', to='specify.collection')),
+                ('createdbyagent', models.ForeignKey(db_column='CreatedByAgentID', null=True, on_delete=specifyweb.specify.models.protect_with_blockers, related_name='+', to='specify.agent')),
+                ('modifiedbyagent', models.ForeignKey(db_column='ModifiedByAgentID', null=True, on_delete=specifyweb.specify.models.protect_with_blockers, related_name='+', to='specify.agent')),
+                ('relativeage', models.ForeignKey(db_column='RelativeAgeID', on_delete=specifyweb.specify.models.protect_with_blockers, related_name='relativeageattachments', to='specify.relativeage')),
+            ],
+            options={
+                'db_table': 'relativeageattachment',
+                'ordering': (),
+            },
+        ),
+        migrations.AddField(
+            model_name='relativeage',
+            name='ageattachment',
+            field=models.ForeignKey(db_column='AgeAttachmentID', null=True, on_delete=specifyweb.specify.models.protect_with_blockers, related_name='relativeages', to='specify.relativeageattachment'),
+        ),
+        migrations.AddField(
+            model_name='relativeage',
+            name='agename',
+            field=models.ForeignKey(db_column='AgeNameID', null=True, on_delete=specifyweb.specify.models.protect_with_blockers, related_name='relativeages', to='specify.geologictimeperiod'),
+        ),
+        migrations.AddField(
+            model_name='relativeage',
+            name='agent1',
+            field=models.ForeignKey(db_column='Agent1ID', null=True, on_delete=specifyweb.specify.models.protect_with_blockers, related_name='+', to='specify.agent'),
+        ),
+        migrations.AddField(
+            model_name='relativeage',
+            name='agent2',
+            field=models.ForeignKey(db_column='Agent2ID', null=True, on_delete=specifyweb.specify.models.protect_with_blockers, related_name='+', to='specify.agent'),
+        ),
+        migrations.AddField(
```

Comment:
Shouldn't it be the opposite? 
model_name='collectionobject',
name='relativeage'

Reply:
now we see CO in the relationships of age tables in schema config but we do not see the age tables in the CO